game/resource/English/ability/units/tooltip_clinkz_strafe_oaa.txt

```diff
@@ -0,0 +1,10 @@
+﻿//=============================================================================
+// Clinkz Strafe
+//=============================================================================
+
+"DOTA_Tooltip_ability_clinkz_strafe_oaa"                            "#{DOTA_Tooltip_ability_clinkz_strafe}"
```

Comment:
Travis is complaining about formatting. But I can't see any issue. My guess is that you put tabs instead of spaces in between or maybe some spurious space somewhere.